### ISSUE TITLE
Migrate to the homebrew bootloader

### ISSUE THREAD

**Issue by SamTebbs33** [SamTebbs33/pluto#66]
It would be nice to at some point use the super-duper bootloader written by @DrDeano . Perhaps with the stretch goal of migrating some of it to Zig if we can (probably a separate issue).

**Comment by SamTebbs33** [comment]
Perhaps we should reconsider this, or at least postpone it. Working on the homebrew bootloader to get it to the same level of functionality as grub would take time that could be spent on other interesting and important tasks

**Comment by DrDeano** [comment]
Yes, this would be a big project, maybe in the future be moved as a separate repo and have it be worked on separately then be integrated when ready. It would be nice to get it working. Might have a look into what needs to be added to the bootloader for it to be usable. Probably step one would be to get it multiboot complaint, well, as least the parts we need.

**Comment by SamTebbs33** [comment]
Agreed. Multiboot compliance (at least what we need of it) would be very nice. Should we close this or keep it open?

**Comment by DrDeano** [comment]
Keep open for now so we don't for get :P Then when we have a organisation and have this is a repo, then can close.
